scripts/collectIncludes.gradle

```diff
@@ -12,7 +12,26 @@ task collectIncludes(
     doLast {
         //let's search the whole project for files, not only the docs folder
         //could be a problem with node projects :-)
-        File sourceFolder = projectDir
+        if (docDir.startsWith('.')) {
```

Comment:
hm. don't we already fix this in the `Asciidoctorbasics.gradle`-file? I have to check this...